test/node.test.js

```diff
@@ -343,4 +343,12 @@ test('supports range selection', () => {
   )
 })
 
+test('supports last versions for Node.js', () => {
+  is(browserslist('last 2 node versions').length >= 1, true)
+})
+
+test('supports last major versions for Node.js', () => {
+  is(browserslist('last 2 node major versions').length >= 1, true)
```

Comment:
This test doesn’t test what Browserslist return. What if it return `node 4` or invalid `node2`?

We need to mock data.,\

Reply:
That `node.test.js` isn't using mock data. Should we create mock data for all tests in that file or just these two new tests?

Reply:
I think it will be better to have 2 types of test:

1. With real data (in case if npm package will break API)
2. With mock data

So let’s keep this `.length` test and add an 2 extra tests with mock data.

Reply:
How can we mock data?

Reply:
Have no idea. What do you suggest?

Reply:
I also don't know how to do, since uvu doesn't support mocking.

Reply:
You can use some another tool for mocking or change Browserslist API to add mocking feature like we do with browser data